content/concepts/the-rippled-server/rippled-server-modes.md

```diff
@@ -2,61 +2,83 @@
 
 The `rippled` server software can run in several modes depending on its configuration, including:
 
-* Stock server - follows the network with a local copy of the ledger.
-* Validating server, or _validator_ for short - participates in consensus (and does everything a stock server does, too).
-* `rippled` server in stand-alone mode - for testing. Does not communicate to other `rippled` servers.
+- [**Network Mode**](#network-mode) - This is the main mode of the server: it follows the peer-to-peer network, processes transactions, and maintains some amount of [ledger history](ledger-history.html). This mode can be configured to do any or all of the following roles:
```

Comment:
I like the name `Network Mode`. However, this is the first time I am seeing this name, so I think we should try to get consensus from the team on whether this is the name we want before we commit to it.